apps/web/src/pages/EmploymentEquityPage/EquityOptions.tsx

```diff
@@ -179,12 +199,12 @@ const EquityOptions: React.FC<EquityOptionsProps> = ({
                   title={intl.formatMessage(getEmploymentEquityGroup("woman"))}
                 />
               )}
-              {!resolvedIndigenous && (
-                <EquityOption
+              {!resolvedIndigenousCommunities.length && (
```

Comment:
See previous comment about array length checks.

Reply:
Fixed in [fixes from Eric's review](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5461/commits/e7a019aa742654d22f4aa3f02ab5dd1297ae656c).